backend/app/tests/api/routes/test_prompts.py

```
@@ -0,0 +1,251 @@
+    non_existing_id = get_non_existent_id(db, Prompt)
+    response = client.patch(
+        f"/api/v1/prompt/{non_existing_id}",
+        headers={"X-API-KEY": user_api_key.key},
+        params={"name": "new_name"},
+    )
+
+    assert response.status_code == 404
+    assert response.json()["error"] == "Prompt not found."
+
+
+from app.models import PromptCreate
```

Ruff (0.12.2)

Module level import not at top of file

(E402)